web.go

```diff
@@ -73,7 +74,7 @@ var cachableFileExtensions = []string{
 	".jpg", ".jpeg", ".png", ".svg", ".gif", ".webp", ".ico", ".woff2",
 }
 
-func setupRouter() *gin.Engine {
+func setupRouter(secureRedirect bool) *gin.Engine {
```

Comment:
Let's not pass this in... should just test `config.TLSEnforce` at [line 87](https://github.com/jetkvm/kvm/pull/1093/changes#diff-a0a6130738465fc41d08d7b30558d8f1e63d7ac34466343db619b1d020487225R87)